langchain4j/src/main/java/dev/langchain4j/chain/ConversationalRetrievalChain.java

```diff
@@ -52,7 +52,7 @@ public String execute(String question) {
 
         question = ensureNotBlank(question, "question");
 
-        List<TextSegment> relevantSegments = retriever.findRelevant(question);
+        List<TextSegment> relevantSegments = retriever.findRelevant("default",question);
```

Comment:
why is this hardcoded?

Reply:
The reason I hardcoded this part is that we need a default message queue.